src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -281,9 +281,9 @@ Convolution::Convolution(const std::shared_ptr<ngraph::Node>& op, const dnnl::en
     }
 
     // Due to performance issue, brgconv will only be enabled by default:
-    // 1, support amx
+    // 1, support avx512
     // 2, static shape(dynamic shape may change weights layout if the input shape changes and cause performance issue: 86948)
-    shouldTryBrgconv = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx) && !isDynamicNode();
+    shouldTryBrgconv = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) && !isDynamicNode();
```

Comment:
I don't think we should change the condition in that way. Let me propose the following:
1. Lets put all brgemm enabling logic in one place. Maybe separate function that will be called inside getSupportedDescriptors().
2. Base condition for shouldTryBrgconv should stay the same: we still not ready to switch on brgconv by default on AVX512 systems since FP32 perf is bad.
3. After shouldTryBrgconv is initialized you can adjust the value for case: AVX512 ISA && INT8 && no binary post ops.

Reply:
Merged to one function. Default:
1, amx enable all brgconv
2, avx512 only enable int8 without binary postops.